dev-python/fuse-python/fuse-python-0.3.1.ebuild

```diff
@@ -0,0 +1,25 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python2_7 python3_{5,6,7} pypy )
+
+inherit distutils-r1
+
+KEYWORDS="~amd64 ~x86"
+DESCRIPTION="Python FUSE bindings"
+HOMEPAGE="https://github.com/libfuse/python-fuse"
+
+SRC_URI="https://github.com/libfuse/python-fuse/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+S="${WORKDIR}/python-fuse-${PV}"
+
+LICENSE="GPL-2"
+SLOT="0"
+IUSE=""
+
+RDEPEND=">=sys-fs/fuse-2.1:0="
+
+DEPEND="${RDEPEND}"
```

Comment:
Python packages rely on setuptools to build. This dependency must be added to `DEPEND`. Please modify this line:
```
DEPEND="
    ${RDEPEND}
    dev-python/setuptools[${PYTHON_USEDEP}]"
```